lexus_rx_450h_2019/drivers.launch.py

```diff
@@ -135,10 +135,8 @@ def generate_launch_description():
             PushRosNamespace(EnvironmentVariable('CARMA_INTR_NS', default_value='hardware_interface')),
             IncludeLaunchDescription(
                 PythonLaunchDescriptionSource([ FindPackageShare('lightbar_driver'), '/launch/lightbar_driver_node_launch.py']),
-                launch_arguments = { 
+                launch_arguments = {
                     'log_level' : GetLogLevel('lightbar_driver', env_log_levels),
-                    'ip_addr' : '192.168.88.28',
-                    'port' : '80',
                     'vehicle_calibration_dir' : vehicle_calibration_dir,
```

Comment:
Calibration folder is also not the input, so should be removed. I know there is auth folder but that is different path

Reply:
And I also don't see the subsystem change as it is required